tests/Test/ContaoManager/PluginTest.php

```diff
@@ -53,6 +54,9 @@ public function testGetBundles()
             )
             ->setReplace(['metamodelsattribute_geodistance']);
 
-        $this->assertArraySubset($plugin->getBundles($parser), [$bundleConfig]);
+        $this->assertTrue(
```

Comment:
This is checking the same condition twice? Why?

Reply:
Fixed